### DISCUSSION TITLE
v2.0.0 - Old Habits Die Hard

### DISCUSSION THREAD

**Discussion by 0xCUB3** [2009-Nissan-Cube/About-This-Hack#97] [Announcements]
## What's Changed
* Added support for macOS 15 Sequoia 🌳
* Added new M3 Macs
* Fixed 3-year-old display centering bug 🐛
* Replaced 98% of shell commands with native Swift code!!! 🎉
* Parallelized some data collection code 🔗
* Fixed broken updates ❤️‍🩹
* Added better command to get display names. By @weachy in https://github.com/0xCUB3/About-This-Hack/pull/95 

## New Contributors
* @weachy made their first contribution in https://github.com/0xCUB3/About-This-Hack/pull/95

**Full Changelog**: https://github.com/0xCUB3/About-This-Hack/compare/1.2.0...1.3.0

<hr /><em>This discussion was created from the release <a href='https://github.com/0xCUB3/About-This-Hack/releases/tag/1.3.0'>v2.0.0 - Old Habits Die Hard</a>.</em>

**Comment by JohnD0406** [comment]
The bootloader used to show the OpenCore version, but now shows "Apple UEFI" like a Mac without OpenCore. I'm running a 2013 Mac Pro with OCLP 1.5.0 & Sonoma 14.5. Was that intentional? Thanks.

**Comment by 0xCUB3** [comment]
that was not intentional. I rewrote some of the code for OpenCore detection which might have broken it for some users. Could you open an issue and paste the Terminal output when dragging the About This Hack binary into a new terminal window?

**Comment by 0xCUB3** [comment]
@JohnD0406 I think I fixed the issue. I uploaded the new version to the release: https://github.com/0xCUB3/About-This-Hack/releases/download/2.0.0/About.This.Hack.zip

Please let me know if it shows up correctly now. I apologize for the issue as I no longer have a Hackintosh/OCLP device to test on :)

**Reply by JohnD0406** [reply]
That fixed it, thanks!

**Comment by semakj** [comment]
When hovering over "Version" the tool tip always shows "System Integrity Protection: Enabled (0x00000000)" although it is Disabled. In my case the OCLP setting is 0x803. Moreover, a Line Feed is missing before "System ..."

**Comment by 0xCUB3** [comment]
@semakj this has been an issue for awhile. I'm looking into it
